src/unraphael/dash/pages/4_compare.py

```diff
@@ -181,50 +224,89 @@ def alignment_help_widget():
     )
 
 
+# def animate_images(img_aligned, baseline, num_frames=100):
+#     fig, ax = plt.subplots()
+#     blended_image = cv2.addWeighted(img_aligned, 1, baseline, 0, 0)
+#     im = ax.imshow(cv2.cvtColor(blended_image, cv2.COLOR_BGR2RGB), extent=[0, blended_image.shape[1], 0, blended_image.shape[0]])
+
+#     def update(frame):
+#         alpha = frame / num_frames
+#         blended_image = cv2.addWeighted(img_aligned, 1 - alpha, baseline, alpha, 0)
+#         im.set_array(cv2.cvtColor(blended_image, cv2.COLOR_BGR2RGB))
+#         ax.set_title(f'Frame {frame + 1}/{num_frames}')
+                           
+#     ani = animation.FuncAnimation(fig, update, frames=num_frames, interval=25, repeat=True, repeat_delay=1000)
+#     return ani.to_jshtml()
+                
+
 def comparison_widget(
```

Comment:
Why is this function being modified?

Reply:
To work with a previous and next button in place of a dropdown menu